health/api/src/main/java/com/cerner/beadledom/health/dto/BuildDto.java

```diff
@@ -15,8 +17,9 @@
  *
  * @since 1.4
  */
-@ApiModel(description = "Indicates the list of dependencies of this service.")
 @AutoValue
+@ApiModel(description = "Indicates the list of dependencies of this service.")
```

Comment:
Make sure the order of these annotations is consistent for all the health DTOs. It seems like most had `JsonDeserialize` before `ApiModel`.

Reply:
Fixed ordering, https://github.com/cerner/beadledom/pull/100/commits/16bb80cf2621d60970f0ad16e06a7708768dd997